src/components/feature-board-file-element/content/display/file-description/FileDescription.vue

```diff
@@ -1,35 +1,34 @@
 <template>
-	<div class="grey lighten-4">
-		<v-app-bar v-if="showTitle" flat color="transparent">
-			<v-toolbar-title data-testid="file-description-toolbar" inactive>
-				<v-icon
-					color="black"
-					class="mr-2"
-					size="18"
-					data-testid="file-description-icon"
-					>{{ mdiFileDocumentOutline }}</v-icon
-				>
+	<div>
+		<ContentElementBar
+			v-if="showTitle"
+			:hasGreyBackground="!isEditMode"
+			:icon="mdiFileDocumentOutline"
+		>
+			<template #title>
+				<ContentElementTitle>{{ name }}</ContentElementTitle>
+			</template>
 
-				<span
-					class="subtitle-1 font-weight-bold"
-					data-testid="file-description-name"
-					>{{ name }}</span
-				>
-			</v-toolbar-title>
+			<template #menu>
+				<slot></slot>
+			</template>
 
-			<v-spacer></v-spacer>
+			<template #subtitle>
+				<span v-if="!isEditMode">
+					{{ caption }}
+				</span>
+			</template>
+		</ContentElementBar>
 
-			<slot></slot>
-		</v-app-bar>
-
-		<v-card-subtitle v-if="caption && !isEditMode">
-			{{ caption }}
-		</v-card-subtitle>
+		<div v-else-if="caption && !isEditMode" class="px-4 py-4 grey lighten-4">
```

Comment:
Both paddings can be combined to :
```suggestion
		<div v-else-if="caption && !isEditMode" class="pa-4 grey lighten-4">
```